full-stack-tests/rpc-interface/src/frontend/setup/IModelSession.ts

```diff
@@ -54,11 +55,21 @@ export class IModelSession {
       iTwinId = iModelData.iTwinId!;
 
     if (iModelData.useName) {
-      const imodelClient = new IModelHubClient();
-      const imodels = await imodelClient.iModels.get(requestContext, iTwinId, new IModelQuery().byName(iModelData.name!));
-      if (undefined === imodels || imodels.length === 0)
+      const imodelClient = new IModelsClient({ api: { baseUrl: `https://${process.env.IMJS_URL_PREFIX ?? ""}api.bentley.com/imodels`}});
+      const imodels = imodelClient.iModels.getRepresentationList({
+        authorization: AccessTokenAdapter.toAuthorizationCallback(await IModelApp.getAccessToken()),
+        urlParams: {
+          projectId: iTwinId,
+        },
+      });
+      for await (const iModel of imodels) {
```

Comment:
Could be filtered by [name](https://github.com/iTwin/imodels-clients/blob/main/clients/imodels-client-management/src/operations/imodel/IModelOperationParams.ts#L22) when querying all iModels in the iTwin. Then we wouldn't need to filter on the client side

Reply:
thanks, just updated